Classes/Core/ConsoleBootstrap.php

```diff
@@ -158,8 +158,12 @@ protected function ensureRequiredEnvironment()
             echo 'The command line must be executed with a cli PHP binary! The current PHP sapi type is "' . PHP_SAPI . '".' . PHP_EOL;
             exit(1);
         }
-        ini_set('memory_limit', -1);
-        set_time_limit(0);
+        if (@ini_get('memory_limit') !== '-1' && @ini_set('memory_limit', '-1') === false) {
+            echo 'Unable to set memory_limit to -1.';
```

Comment:
a plain echo at this point is exactly what I wanted to avoid :)

Please just do `ini_get('memory_limit') !== '-1' && @ini_set('memory_limit', '-1')` (note that we do not need to use the shutup operator on ini_get)